### ISSUE TITLE
Optionally embed (or just link to) an etherpad for each session

### ISSUE THREAD

**Issue by toolness** [OpenNews/srccon-schedule#1]
During SRC CON, each session seemed to have its own etherpad for note-taking, like [the etherpad for Jeff Larson's security talk](https://etherpad.mozilla.org/srcsecurity). I thought it might be neat if each etherpad were linked to from the schedule details page, or even embedded via IFRAME so people didn't have to share an extra link around before each session started, and so the notes wouldn't potentially get lost after the session ended.

Just a thought, feel free to close this issue if it's out of scope!

**Comment by ryanpitts** [comment]
Yep, this is a great idea. There were only a couple sessions with repo and/or etherpad links before the conference actually started, but many more popped up as the conference moved along. Probably should handle this with new attribute(s) in the session json.

**Comment by toolness** [comment]
Ah that's cool! Yeah, I had forgotten about the repos, like @zzolo's [style guide discussion](https://github.com/zzolo/srccon-style-guide-discussion). Would be so cool to link to that, maybe even embed the README.md in the details page for the session!
